phases/06-speech-and-audio/16-anti-spoofing-audio-watermarking/code/main.py

```
@@ -0,0 +1,139 @@
+    print(f"  fake  mean: {sum(fake_scores)/len(fake_scores):.3f}")
+
+    print()
+    print("=== Step 3: sweep threshold → EER ===")
+    candidates = sorted(set(real_scores + fake_scores))
+    best = (1.0, 0.0, 0.0, 0.0)
+    for t in candidates:
+        far = sum(1 for s in fake_scores if s >= t) / len(fake_scores)
+        frr = sum(1 for s in real_scores if s < t) / len(real_scores)
+        if abs(far - frr) < best[0]:
+            best = (abs(far - frr), t, far, frr)
+    gap, t, far, frr = best
```

Ruff (0.15.10)

[warning] Unpacked variable `gap` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)